src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -424,6 +424,30 @@ make_coordinate_map_base(Maps&&... maps) noexcept {
       std::forward<Maps>(maps)...);
 }
 
+/// \ingroup ComputationalDomainGroup
+/// \brief Creates a std::vector<std::unique_ptr<CoordinateMapBase>
+/// containing the result of make_coordinate_map_base applied to each
+/// argument passed in.
+template <typename SourceFrame, typename TargetFrame, typename Arg1,
+          typename... Args>
+std::vector<std::unique_ptr<
+    CoordinateMapBase<SourceFrame, TargetFrame, std::decay_t<Arg1>::dim>>>
+make_vector_coordinate_map_base(Arg1&& arg_1,
+                                Args&&... remaining_args) noexcept {
+  std::vector<std::unique_ptr<
+      CoordinateMapBase<SourceFrame, TargetFrame, std::decay_t<Arg1>::dim>>>
+      return_vector;
+  return_vector.reserve(sizeof...(Args) + 1);
+  return_vector.emplace_back(make_coordinate_map_base<SourceFrame, TargetFrame>(
+      std::forward<Arg1>((arg_1))));
```

Comment:
I know you like using extra parentheses to clarify things (which I'm generally fine with), but this seems a bit silly.